DotNetHowToSynonyms/DotNetHowToSynonyms/DotNetHowToSynonyms.csproj

```diff
@@ -0,0 +1,89 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{001AB02D-B165-4FB4-808C-48502C69E8BA}</ProjectGuid>
```

Comment:
Thanks. Generated a GUID and updated the two files. 

---
In reply to: [111008524](https://github.com/Azure-Samples/search-dotnet-getting-started/pull/10#discussion_r111008524) [](ancestors = 111008524)